WORKSPACE

```diff
@@ -5,14 +5,28 @@ workspace(
     name = "aspect_rules_js",
 )
 
-load(":internal_deps.bzl", "rules_js_internal_deps")
+load(":internal_deps.bzl", "js_internal_deps")
 
-rules_js_internal_deps()
+js_internal_deps()
 
 # Install our "runtime" dependencies which users install as well
-load("//js:repositories.bzl", "rules_js_dependencies")
+load("//js:repositories.bzl", "js_dependencies")
 
-rules_js_dependencies()
+js_dependencies()
+
+load("//js:configure.bzl", "js_configure")
+
+js_configure()
+
+load("@aspect_rules_js//js:npm_import.bzl", "npm_import", "translate_pnpm_lock")
+
+translate_pnpm_lock(
+    name = "aspect_rules_js_lifecycle_npm_deps",
+    # Avoid a circular dependency on the lifecycle hooks package,
+    # which is used when enable_lifecycle_hooks = True
+    enable_lifecycle_hooks = False,
+    pnpm_lock = "@aspect_rules_js//js/private/lifecycle:pnpm-lock.json",
+)
```

Comment:
We vendor the files from the repo this creates in `//js/private/lifecycle`, and in `js_configure` we load the `npm_repositories` to install the lifecycle deps.